nifty-ssl/src/main/java/com/facebook/nifty/ssl/CryptoUtil.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2012-2013 Facebook, Inc.
```

Comment:
This is enforced by a maven plugin. If I change the plugin parameters, then I have to change the year to 2016 in all existing files. Should be done in a separate diff.